spi/spi_master_slave/README.md

```diff
@@ -0,0 +1,98 @@
+# spi_master and spi_slave
```

Comment:
Would it be possible to provide a README.adoc file? For example see https://github.com/raspberrypi/pico-examples/tree/master/spi/bme280_spi. If so we can automatically pull this and the example code into the SDK book into the App Note appendix.

Reply:
Sure. 🙂

Reply:
Added, resolving.